Notify/Notify/Notify/AppShell.xaml.cs

```diff
@@ -154,22 +157,28 @@ private void setMessagingCenterLocationSubscription()
             });
         }
 
-        private static void updateStatusOfSentNotifications(List<Notification> arrivedLocationNotifications)
+        private static void updateStatusOfSentNotifications(params List<Notification>[] sentNotificationsArrays)
         {
             string json = Preferences.Get(Constants.PREFERENCES_NOTIFICATIONS, string.Empty);
             List<Notification> notifications = JsonConvert.DeserializeObject<List<Notification>>(json);
-            
+
             notifications.ForEach(notification =>
             {
-                if (arrivedLocationNotifications.Any(arrivedNotification => arrivedNotification.ID.Equals(notification.ID)))
+                foreach (List<Notification> sentNotifications in sentNotificationsArrays)
```

Comment:
sentNotificationsList and sentNotificationLists would be a better naming in this issue, don't you think?